### PR TITLE
xh: Add version 0.7.0

### PR DESCRIPTION
**Pull request by myty**
Adds a manifest for https://github.com/ducaale/ht

### CODE REVIEW

**Comment by ducaale** [comment]
@myty ht author here, Would you mind renaming `ht-rs` to be consistent with the [homebrew-core](https://github.com/Homebrew/homebrew-core/blob/master/Formula/ht-rust.rb) version which is now called ht-rust? Thank you very much!

**Comment by myty** [comment]
> @myty ht author here, would you mind renaming `ht-rs` to be consistent with the [homebrew-core](https://github.com/Homebrew/homebrew-core/blob/master/Formula/ht-rust.rb) version which is now called ht-rust? Thank you very much!

For sure!

**Comment by myty** [comment]
> [Your changes do not pass checks.](https://github.com/Ash258/Scoop-GithubActions/wiki/Pull-Request-Checks)
> 
> ### ht-rust
> * [x]  Description
> * [x]  License
> * [x]  Hashes
> * [ ]  Checkver
> * [x]  Autoupdate

Not sure why Checkver is failing.  I'm going to run verify again. 🤷🏻‍♂️

File: bucket/ht-rust.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "homepage": "https://github.com/ducaale/ht/",
+    "description": "Yet another HTTPie clone in Rust.",
+    "license": "MIT",
+    "version": "0.5.0",
```
Comment (earnestma) [inline bucket/ht-rust.json:5]: Formatting

```suggestion
    "version": "0.5.0",
    "description": "Yet another HTTPie clone in Rust",
    "homepage": "https://github.com/ducaale/ht",
    "license": "MIT",
```

File: bucket/ht-rust.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "url": "https://github.com/ducaale/ht/releases/download/v0.5.0/ht-x86_64-pc-windows-msvc.zip",
+    "hash": "eca94991b75ac4cb718f6c1449ae521ea692c73ab5ee1ff326e28636f3314c1a",
```
Comment (earnestma) [inline bucket/ht-rust.json:7]: afaik x64_64 means 64-bit only:

```suggestion
    "architecture": {
        "64bit": {
            "url": "https://github.com/ducaale/ht/releases/download/v0.5.0/ht-x86_64-pc-windows-msvc.zip",
            "hash": "eca94991b75ac4cb718f6c1449ae521ea692c73ab5ee1ff326e28636f3314c1a"
        }
    },
```

File: bucket/ht-rust.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "bin": "ht.exe",
+    "checkver": {
+        "github": "https://github.com/ducaale/ht"
+    },
```
Comment (earnestma) [inline bucket/ht-rust.json:11]: When homepage is same as Github repo link, there is no need to specify twice

```suggestion
    "checkver": "github",
```

File: bucket/ht-rust.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    "autoupdate": {
+        "url": "https://github.com/ducaale/ht/releases/download/v$version/ht-x86_64-pc-windows-msvc.zip"
+    }
+}
```
Comment (earnestma) [inline bucket/ht-rust.json:15]: Autoupdate:

```suggestion
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/ducaale/ht/releases/download/v$version/ht-x86_64-pc-windows-msvc.zip"
            }
        }
    }
}
```

**Review by earnestma — CHANGES_REQUESTED** [review]

**Comment by ducaale** [comment]
v0.7.0 of `ht` has been renamed to `xh` due to name collision with `/usr/bin/ht` from TextLive. See https://github.com/ducaale/xh/issues/1

**Review by r15ch13 — APPROVED** [review]

**Pull request merged**